File: go/P0002/sum-even-valued-fibo/main.go
package main

import (
	"flag"
	"fmt"
)

var (
	below = flag.Uint64("below", 0, "max Fibonacci number to stop the sum of the even terms to")
)

func main() {
	flag.Parse()
	ans := sumEvenFibonacciUntil(*below)
	fmt.Println("The sum of the even-valued terms of the Fibonacci sequence below", *below, "is:", ans)
}

File: rust/P0002/sum-even-valued-fibo/src/main.rs
use structopt::StructOpt;

mod solution;
use solution::sum_even_valued_fibo_below;


#[derive(StructOpt)]
struct Cli {
    below: u128
}

fn main() {
    let args = Cli::from_args();
    let ans = sum_even_valued_fibo_below(args.below);
    println!("TThe sum of the even-valued terms of the Fibonacci sequence below {0} is: {1}", args.below, ans);
}
